shared/bash_remediation_functions/rhel7_handle_ntp_like_file.sh

```diff
@@ -0,0 +1,16 @@
+# Function ensures that the ntp/chrony config file contains valid server entries
+# $1: Path to the config file
+# $2: Comma-separated list of servers
+function rhel7_handle_ntp_like_file {
+    # If invoked with no arguments, exit. This is an intentional behavior.
+    [ $# -gt 1 ] || return 0
+    [ $# = 2 ] || die "$0 requires zero or exactly two arguments"
+	local _config_file="$1" _servers_list="$2"
+	if ! grep -q '#[[:space:]]*server' "$_config_file"; then
+		for server in $(echo "$_servers_list" | tr ',' '\n') ; do
+			printf '\nserver %s iburst' "$server" >> "$_config_file"
+		done
+	else
+		sed -i 's/#[ \t]*server/server/g' "$_config_file"
```

Comment:
This remediation looks misleading:
If the file doesn't have any `server` entry, the list of servers from variable received is added.
If there are commented `server` entries, the remediation just uncomments it, and no check is done regarding the servers itself.

Also, OVAL definition doesn't care about which servers are configured.
If we set the variable for the list of servers, we might as well be checking for them.
For this PR we can keep original behavior, but we can improve it according to expectations on variable.